### ISSUE TITLE
crawler container can't see rethink on localhost

### ISSUE THREAD

**Issue by abitrolly** [cyberFund/cybernode-archive#57]
An issue for the record. Docker containers are unable to connect to host `localhost` services.

I started RethinkDB and forwarded ports 8080 and 28015.
```
docker run --name rethink -p 127.0.0.1:8080:8080 -p 127.0.0.1:28015:28015 rethinkdb:2.3
#    -p localhost:8080:8080  - host:container - expose port 8080 as 127.0.0.1:8080 on host
```
RethinkDB is listening and I can forward and check admin interface at http://127.0.0.1
```
ssh -v -NL 127.0.0.1:8080:127.0.0.1:8080 mars.cybernode.cyber.fund
```

But `crawler` is unable to connect to `localhost:28015` or `127.0.01:28015`, because it runs in isolated network stack. I don't know why I thought it would be capable. Perhaps because docker processes are visible in `ps faux` and that they are able to access `docker0` interface without any ports being published (https://github.com/cyberFund/cybernode/issues/51#issuecomment-309387875).

https://github.com/cyberFund/xchange-crawler/issues/37 and https://github.com/cyberFund/xchange-crawler/issues/38 delayed troubleshooting.

**Comment by abitrolly** [comment]
Because it is impossible to forward port **from** host to container, the only solution here is to get ip address of RethinkDB container in `docker0` network after it is executed and feed it to xchange-crawler.
```
docker inspect --format='{{.NetworkSettings.IPAddress}}' rethink
```

**Comment by abitrolly** [comment]
Temporarily fixed in #58.
